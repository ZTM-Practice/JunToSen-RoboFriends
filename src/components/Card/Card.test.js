import { shallow } from 'enzyme';
import React from 'react';
import Card from './Card';

describe('Card Component', () => {
    it('expect to render the Card component', () => {
        const mockName = 'R2-D2';
        const mockEmail = 'beepboop@gmail.com';
        const mockId = 5;
        expect(shallow(<Card name={mockName} email={mockEmail} id={mockId} />)).toMatchSnapshot();
    });
});
