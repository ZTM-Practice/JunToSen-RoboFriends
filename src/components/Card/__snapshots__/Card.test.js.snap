// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component expect to render the Card component 1`] = `ShallowWrapper {}`;
